connectivity/suite.go

```diff
@@ -240,6 +275,124 @@ func Run(ctx context.Context, ct *check.ConnectivityTest) error {
 			return check.ResultDrop, check.ResultNone
 		})
 
```

Comment:
What's the difference in this test vs. the previous one? Both seem to be basic label selection for denies, it's only the content of the labels themselves which is different. This seems like it's already covered by `client-ingress-from-client2-deny`?

Reply:
^ This was meant to comment on `clientIngressFromClient2DenyPolicyYAML` vs. `echoIngressFromOtherClientDenyPolicyYAML`

Reply:
good catch, let me keep echoIngressFromOtherClientDenyPolicyYAML as this test will test same + cross node traffic.